clients/fides-js/src/lib/hooks/useUUID4.ts

```diff
@@ -0,0 +1,14 @@
+import { useState } from "preact/hooks";
+import { v4 as uuidv4 } from "uuid";
+
+/**
+ * Custom hook that generates a UUIDv4.
+ * The returned value stays the same for the lifetime of the component.
+ * @returns The generated UUIDv4.
+ */
+const useUUID4 = () => {
+  const [uuid] = useState<string>(uuidv4());
+
+  return uuid;
+};
+export default useUUID4;
```

Comment:
This doesn't appear to me to get imported anywhere, is it still needed?

Reply:
Good catch, it's not needed. It was used in an earlier approach, but then had to change it. I left it as it might be useful for someone,  but I can easly remove it too.

Reply:
If you think it'll still be useful in the future I don't have a problem leaving it in, just wanted to make sure it wasn't a total oversight!